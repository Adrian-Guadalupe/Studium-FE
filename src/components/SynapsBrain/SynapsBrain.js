import React from 'react';
import { ReactComponent as SynapsBrainImage } from '../../images/Frame.svg';
import PropTypes from 'prop-types';
import { SvgContainer} from '../SvgContainer/SvgContainer.js';

/**
 *  Synaps Brain SVG
 *
 * @description Displays a svg of the synaps brain adjustable styles.
 *
 *  @component
 *  @example
 *  return (
 *    <SynapsBrain
 *       svg={svg}
 *       zIndex={ 1 }
 *       position={ 'absolute' }
 *       backgroundColor={ theme.primaryColor }
 *       color={ '#164172' }
 *       opacity={ 1 }
 *       strokeColor={ theme.primaryColor }
 *       viewBox={ '225 25 400 400' }
 *    />
 *    )
 */
export const SynapsBrain = ( props ) => {
  
  return <SvgContainer svg={ SynapsBrainImage } { ...props }/>;
};

SynapsBrain.propTypes = {
  position: PropTypes.string,
  left: PropTypes.string,
  top: PropTypes.string,
  backgroundColor: PropTypes.string,
  opacity: PropTypes.number,
  color: PropTypes.string,
  strokeColor: PropTypes.string,
  stokeWidth: PropTypes.string,
  viewBox: PropTypes.string,
  zIndex: PropTypes.number,
};
