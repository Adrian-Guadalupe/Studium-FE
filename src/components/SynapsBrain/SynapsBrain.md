```js
import {theme} from '../../utilities/theme.js';
import {ContainerDiv} from '../Container/ContainerDiv.js';
<ContainerDiv position="relative" width={"300px"} height="300px">
    <SynapsBrain
              zIndex={ 1 }
              position={ 'absolute' }
              backgroundColor={ theme.primaryColor }
              color={ '#164172' } opacity={ 1 }
              strokeColor={ theme.primaryColor }
              viewBox={ '225 25 400 400' }
            />
    <h2 style={{zIndex: 3, color: "white"}}>Text</h2>
</ContainerDiv>
```