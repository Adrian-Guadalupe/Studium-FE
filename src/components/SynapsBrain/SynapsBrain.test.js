import React from 'react';
import renderer from 'react-test-renderer';

import { SynapsBrain } from './SynapsBrain.js';

test( 'StyledCardDeck render correctly', () => {
  const tree = renderer.create( <SynapsBrain/> ).toJSON();
  expect( tree ).toMatchSnapshot();
} );