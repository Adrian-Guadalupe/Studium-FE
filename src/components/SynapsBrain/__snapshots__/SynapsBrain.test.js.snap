// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Synaps Brain renders correctly 1`] = `
<div>
   
  <h1>
    Synaps Brain
  </h1>
   
</div>
`;
